src/style.css

```diff
@@ -0,0 +1,171 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-size: 1.3rem;
+  font-family: 'Inconsolata', monospace;
+  font-weight: 300;
+  outline: none;
+}
+
+body {
+  display: flex;
```

Comment:
Since you are using several display flex declarations would be great in order to improve KISS that you can use a class that has already been set up as flex and then add it to your different element selectors in HTML.